src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,9 +5,24 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bladeWidth;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. It's better to make `bladeWidth` private and access it through getters and setters.